assets/prefabs/Gojiberry.prefab

```diff
@@ -0,0 +1,10 @@
+{
+  "parent": "engine:iconItem",
+  "DisplayName": {
+    "name": "Gojiberry"
+  },
+  "Item": {
+    "icon": "SimpleFarming:SimpleFarming#Gojiberry",

```

Comment:
The icon won't load this way and resolve in the placeholder texture.
The reason is, that you define your texture in the textures folder (which is correct), whereas the testberry texture is defined in `assets/atlas/SimpleFarming.atlas` which splits the `assets/textures/SimpleFarming.png` multi texture.
In your case, the icon `SimpleFarming:SimpleFarming#Gojiberry` will resolve to _Fetch the resource from SimpleFarming_ `:` _Look for a resource named SimpleFarming_ (the atlas, as it matches for an icon) `#` _then resolve the Atlas enry for Gojiberry_ (it does not exist).

Correct would be
`SimpleFarming:Gojiberry`
Which would resolve to: _Fetch a resource from SimpleFarming_ `:` _and use the gojiberry texture_ (matches for an icon)